.github/workflows/stale.yaml

```diff
@@ -0,0 +1,46 @@
+name: Mark and Close Stale Issues and PRs
+
+on:
+  schedule:
+    - cron: '0 0 * * *' # runs every day at midnight UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Mark and close stale issues and PRs
+        uses: actions/stale@v9
+        with:
+          stale-issue-label: 'lifecycle/stale'
+          exempt-issue-labels: 'reminder'
```

Comment:
exempt-issue-labels can be `kind/feature, kind/bug, help wanted, good first issue`

Reply:
done.